modules/Utilities/src/Utils.cpp

```diff
@@ -7,6 +7,7 @@
  */
 
 // std
+#define _USE_MATH_DEFINES
```

Comment:
To avoid warning when this is already defined at the CMake level for some reason, I suggest to have: 
~~~
#ifndef _USE_MATH_DEFINES
#define _USE_MATH_DEFINES
#endif 
~~~
You can  also guard all this definition in `#ifdef _MSC_VER`, but perhaps it is not necessary.
~~~